themes/hugo-creative-theme/layouts/partials/contact.html

```diff
@@ -0,0 +1,45 @@
+{{ "<!-- CONTACT -->" | safeHTML }}
+<section id="contact">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2 text-center">
+                {{ with .Site.Params.contact.headline }}
+                    <h2 class="section-heading">{{ . }}</h2>
+                {{ end }}
+                {{ if and .Site.Params.contact.headline .Site.Params.contact.description }}
```

Comment:
I don't know the syntax, should be an `and` really be here?

Reply:
This was in the original template; after examining it it seems to be correct...looks like the "and" applies to the two parts following it.